src/main/java/ch/wisv/payments/admin/products/ProductServiceImpl.java

```diff
@@ -130,13 +142,32 @@ public boolean isProductAvailable(Integer productId) {
     }
 
     @Override
-    public void deleteProduct(int productId) {
-        List<Order> orders = orderService.getOrdersByProductId(productId);
+    public ProductGroup getProductGroupById(Integer productGroupId) {
+        return productGroupRepository.findOne(productGroupId);
+    }
 
-        if (orders.size() > 0) {
-            throw new ProductInUseException("Products are already ordered");
+    @Override
+    public void editProductGroup(ProductGroupRequest productGroupRequest) {
+        if (productGroupRequest.getId() != 0) {
+            Committee committee = committeeRepository.findOne(productGroupRequest.getCommitteeId());
+            ProductGroup productGroup = productGroupRepository.findOne(productGroupRequest.getId());
+
+            productGroup.setCommittee(committee);
+            productGroup.setName(productGroupRequest.getName());
+            productGroup.setDescription(productGroupRequest.getDescription());
+            productGroup.setGroupLimit(productGroupRequest.getGroupLimit());
+
+            productGroupRepository.saveAndFlush(productGroup);
+        }
+    }
+
+    @Override
+    public void deleteProductGroup(int productGroupId) {
+        ProductGroup productGroup = productGroupRepository.findOne(productGroupId);
+        if (!productGroup.getProducts().isEmpty()) {
+            throw new ProductGroupInUseException("Product group must be empty");
```

Comment:
I had previously implemented it in a way where products were removed from the group when the group was removed, but this may be an action where we don't want it to be very easy to perform. I'm also not quite sure which way to go, it made sense to me that since there is a limit involved, you don't want it to be this easy, just like removing a product that is already ordered.